kroxylicious-operator/src/main/java/io/kroxylicious/kubernetes/operator/OperatorMain.java

```diff
@@ -53,6 +55,7 @@ public OperatorMain() throws IOException {
 
     @VisibleForTesting
     OperatorMain(@Nullable KubernetesClient kubeClient, @NonNull HttpServer managementServer) {
+        configureDefaultFabric8JsonMapper();
```

Comment:
can remove this, JavaTimeMapper is registered in `io.fabric8.kubernetes.client.utils.KubernetesSerialization#configureMapper`